projects/observability/src/shared/components/card-list/card-list.component.ts

```diff
@@ -42,7 +43,7 @@ export class CardListComponent implements AfterContentInit {
   public selectedCard?: CardContainerComponent;
 
   public onCardClicked(card: CardContainerComponent): void {
-    this.selectedCard = this.selectedCard === card ? undefined : card;
+    this.selectedCard = card.disabled ? this.selectedCard : this.selectedCard === card ? undefined : card;
```

Comment:
nit: Can you could better represent this to reduce the cognitive complexity?

Reply:
done